Subject: Fix test failure when $HOME is undefined
From: Daniel Shahaf <danielsh@apache.org>
Bug: https://github.com/zsh-users/zsh-syntax-highlighting/issues/216
Last-Update: 2015-10-21

This patch is obsoleted by upstream commit b2ac98b98150b097e9a3dbea8a7b02b4a5cfd0bd
(post-0.3.0).
---

Index: z-sy-h/tests/test-highlighting.zsh
===================================================================
--- z-sy-h.orig/tests/test-highlighting.zsh
+++ z-sy-h/tests/test-highlighting.zsh
@@ -69,6 +69,7 @@ for data_file in ${0:h:h}/highlighters/$
   PREBUFFER= BUFFER= ; expected_region_highlight=(); errors=()
   echo -n "* ${data_file:t:r}: "
   ZSH_HIGHLIGHT_STYLES=( "${(@kv)save_ZSH_HIGHLIGHT_STYLES}" )
+  [[ -d $HOME ]] || HOME="." # must be an existing directory for test-data/path-tilde-home.zsh
   . $data_file
 
   # Check the data declares $PREBUFFER or $BUFFER.
